app/main/views/g_cloud.py

```diff
@@ -365,7 +365,7 @@ def save_search(framework_framework):
             form.name.errors = ["Names must be between 1 and 100 characters"]
 
         if not save_search_selection and request.method == "POST":
-            save_search_selection_error = "Please choose where to save your search."
+            save_search_selection_error = "Please choose where to save your search"
```

Comment:
Can we put this content in the template? Generally we try to keep copy there rather than in the code itself.